atlasdb-config/build.gradle

```diff
@@ -37,3 +46,17 @@ dependencies {
     testCompile group: 'com.github.tomakehurst', name: 'wiremock'
     testCompile group: 'org.assertj', name: 'assertj-core'
 }
+
+shadowJar {
```

Comment:
It increases from ~104K to ~134K. I think this is acceptable given that in absolute terms it's very small though.

```
-rw-r--r--   1 jkong  group  106845  7 Apr 11:46 atlasdb-config-0.38.0-28-gc1d6c3a.dirty.jar
-rw-r--r--   1 jkong  group  137042 12 Apr 13:04 atlasdb-config-0.38.0-37-g7ffd54d.dirty.jar
```